tests/test_project/tests/test_api.py

```diff
@@ -3,8 +3,8 @@
 from openwisp_users.tests.utils import TestOrganizationMixin
 from test_project.serializers import ShelfSerializer
 
-from . import CreateMixin
 from ..models import Shelf
+from . import CreateMixin
```

Comment:
Is there any particular reason for this change?

Reply:
There was an error in build-
`ERROR: /home/travis/build/Weirdo914/openwisp-utils/tests/test_project/tests/test_api.py Imports are incorrectly sorted.`